### PR TITLE
transmitter.py documented

### PR DESCRIPTION
**Pull request by SanyaNanda**
I have documented transmitter.py

### CODE REVIEW

File: transmitter.py [hunk 1]
```diff
@@ -8,9 +8,13 @@
 
 
 def encode_byte(x):
+	''' This function encodes the frequency of the audio.
```
Comment (sushrut111) [inline transmitter.py]: At all the places, can you please add return types and parameter types as well?
Please refer to the snapshot I have attached in the issue!
Thanks.

**Review by sushrut111 — CHANGES_REQUESTED** [review]

**Comment by SanyaNanda** [comment]
Sure

File: transmitter.py [hunk 2]
```diff
@@ -21,16 +35,45 @@ def modulate(msg):
...
+
+	# stream is opened
 	stream = p.open(format = p.get_format_from_width(f.getsampwidth()),  
 	                channels = f.getnchannels(),  
 	                rate = f.getframerate(),  
-	                output = True)  
-	data = f.readframes(chunk)  
+	                output = True)
+	''' 
+	:param int format: sampling size and format
```
Comment (sushrut111) [inline transmitter.py]: This is not a function definition.

**Review by sushrut111 — CHANGES_REQUESTED** [review]

File: transmitter.py (hunk 2 above)
Comment (SanyaNanda) [reply transmitter.py]: You want me to remove this multi-line comment?

Comment (sushrut111) [reply transmitter.py]: Yes. I think this reduces readability.

**Review by sushrut111 — APPROVED** [review]

**Comment by sushrut111** [comment]
Thank you for your contribution @SanyaNanda

**Comment by SanyaNanda** [comment]
Thank you so much! This was a great learning experience

**Pull request merged**